apps/survey/contracts/Surveying.sol

```diff
@@ -0,0 +1,256 @@
+pragma solidity 0.4.18;
+
+import "@aragon/os/contracts/apps/AragonApp.sol";
+
+import "@aragon/os/contracts/lib/minime/MiniMeToken.sol";
+import "@aragon/os/contracts/lib/zeppelin/math/SafeMath.sol";
+import "@aragon/os/contracts/lib/zeppelin/math/SafeMath64.sol";
+import "@aragon/os/contracts/lib/misc/Migrations.sol";
+
+
+contract Surveying is AragonApp {
+    using SafeMath for uint256;
+    using SafeMath64 for uint64;
+
+    uint256 constant public PCT_BASE = 10 ** 18; // 0% = 0; 1% = 10^16; 100% = 10^18
+    uint256 constant public NO_VOTE = 0;
+    bytes32 constant public CREATE_SURVEYS_ROLE = keccak256("CREATE_SURVEYS_ROLE");
+    bytes32 constant public MODIFY_PARTICIPATION_ROLE = keccak256("MODIFY_PARTICIPATION_ROLE");
+
+    MiniMeToken public token;
+    uint256 public minAcceptParticipationPct;
+    uint64 public surveyTime;
+
+    struct OptionCast {
+        uint256 optionId;
+        uint256 stake;
+    }
+
+    /* To allow multiple option votes.
```

Comment:
Weird comment format, please start with '/**'

Reply:
Weird?
http://solidity.readthedocs.io/en/v0.4.23/layout-of-source-files.html?highlight=comments#comments